rustfmt.toml

```diff
@@ -1,2 +1,3 @@
 imports_indent = "block"
 merge_imports = true
+blank_lines_upper_bound = 2    # Clearer seperation helps scan-ability
```

Comment:
don't change global formatting rules inside PRs

Reply:
Why not? How am I supposed to change them? This setting is not going to change existing code.